test/requirements.txt

```diff
@@ -4,7 +4,7 @@ boto3==1.34.23
 certifi>=2023.7.22
 invoke==2.2.0
 mock
-pennylane
+pennylane>=0.35.0
```

Comment:
May reason for this is that qml.node has a new import. Using the wrong version will cause failures.